lib/forms/CoBrALab-Mouse-End-Of-Life-Form/index.ts

```diff
@@ -384,11 +352,9 @@ export default defineInstrument({
     terminationType: z.string().optional(),
     surgeryDeathCause: z.string().optional(),
     bloodCollected: z.number().optional(),
-    embryoPresent: z.boolean().optional(),
     gestationalDay: z.date().optional(),
```

Comment:
I think gestational day is a int number, please check with students.

Reply:
After speaking to the researchers i think its best to move gestational day to the animal origin form